AutocompleteClientTests/FW/API/Parser/SearchResponseParserTests.swift

```diff
@@ -43,7 +43,7 @@ class SearchResponseParserTests: XCTestCase {
         do {
             _ = try self.parser.parse(searchResponseData: validJSONData)
             XCTFail(failWithMessage)
-        } catch CIOError.invalidResponse {
+        } catch _ as CIOError {
             XCTAssertTrue(true)
```

Comment:
Let's retain the specificity here `XCTAssertEqual(_.errorType, .invalidResponse`)